ad7768-1fmcz/ad77681.c

```diff
@@ -266,6 +647,8 @@ int32_t ad77681_set_power_mode(struct ad77681_dev *dev,
 				      AD77681_POWER_CLK_PWRMODE_MSK,
 				      AD77681_POWER_CLK_PWRMODE(mode));
 
+	dev->power_mode = mode;
```

Comment:
This seems like a fix. Add a separate commit for it.
If `ad77681_spi_write_mask` fails, is `dev->power_mode` is still set.